test/rekt/features/broker/source_to_sink.go

```diff
@@ -91,73 +95,29 @@ func SourceToSinkWithDLQ(brokerName, sinkName string, prober *eventshub.EventPro
 		panic(fmt.Errorf("can not find event files: %s", err))
 	}
 
-	f.Setup("install trigger", trigger.Install(via, brokerName, trigger.WithSubscriber(nil, "bad://uri")))
-
-	// Resources ready.
-	f.Setup("trigger goes ready", trigger.IsReady(via))
-
-	// Install events after data plane is ready.
-	f.Setup("install source", prober.SenderInstall("source"))
-
-	// After we have finished sending.
-	f.Requirement("sender is finished", prober.SenderDone("source"))
-
-	// Assert events ended up where we expected.
-	f.Stable("broker with DLQ").
-		Must("accepted all events", prober.AssertSentAll("source")).
-		Must("deliver event to DLQ", prober.AssertReceivedAll("source", sinkName))
-
-	return f
-}
+	f.Setup("install dlq", prober.ReceiverInstall(dlq))
 
-// SourceToTwoSinksWithDLQ tests to see if a Ready Broker acts as middleware.
-//
-// source ---> broker +--[trigger<via1>]--> bad uri
-//                |   |
-//                |   +--[trigger<vai2>]--> sink2
-//                |
-//                +--[DLQ]--> sink1
-//
-func SourceToTwoSinksWithDLQ(brokerName, sink1, sink2 string, prober *eventshub.EventProber) *feature.Feature {
+	brokerConfig := append(broker.WithEnvConfig(), delivery.WithDeadLetterSink(prober.AsKReference(dlq), ""))
+	f.Setup(fmt.Sprintf("install broker %q", brokerName), broker.Install(brokerName, brokerConfig...))
```

Comment:
Generated names for test names are a problem in spyglass
```suggestion
	f.Setup("install broker", broker.Install(brokerName, brokerConfig...))
```